### PR TITLE
Fix/settings/use components multiselect

### LINKED ISSUE #14735: Style change of app properties needed
**Is your feature request related to a problem? Please describe.**
With the upgrade from NC 13 to NC15.0.5 I noticed the style change for the properties of the installed apps. The properties are all listed on the right side now.

Anyhow we have two apps, which are restricted to 24 LDAP groups currently. But the new style make it impossible to view the groups on one screen. 
Currently I need a look into the database to get information about group restrictions. :-(

**Describe the solution you'd like**
I don't know. Maybe any kind of possibility to see the whole list of groups to which the app is restricted to.

The following screenshot should help to understand my problem. Every rectangle inside of the text field seems to be a LDAP group:
group_restriction

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #14735 
Capture d’écran_2019-04-02_15-26-43

### CODE REVIEW

File: settings/src/components/appList.vue [hunk 1]
```diff
@@ -73,15 +73,13 @@
 
 <script>
 import appItem from './appList/appItem';
-import Multiselect from 'vue-multiselect';
```
Comment (skjnldsv) [inline settings/src/components/appList.vue]: was not used in this page

File: settings/src/components/appList/appItem.vue [hunk 2]
```diff
@@ -61,7 +61,6 @@
 </template>
 
 <script>
-	import Multiselect from 'vue-multiselect';
```
Comment (skjnldsv) [inline settings/src/components/appList/appItem.vue]: was not used in this page

**Review by ChristophWurst — APPROVED** [review]
Makes sense :+

**Review by MorrisJobke — APPROVED** [review]
tested and works 👍

**Comment by MorrisJobke** [comment]
@skjnldsv Mind to open the backport?

**Comment by skjnldsv** [comment]
I don't want to do it. This is not a security fix nor a design fix, this is a feature on the components. We should not backport :)

**Comment by MorrisJobke** [comment]
Okay :)

**Pull request merged**